fn/result.go

```diff
@@ -81,11 +81,23 @@ func (r Result[T]) Option() Option[T] {
 	return r.Either.LeftToOption()
 }
 
+// Hush mutes the error value of the result.
+func (r Result[T]) Hush() Option[T] {
+	return r.Either.LeftToOption()
+}
+
 // WhenResult executes the given function if the Result is a success.
+//
+// Deprecated: Use WhenOk instead.
 func (r Result[T]) WhenResult(f func(T)) {
```

Comment:
Remove rather than deprecate with a comment?

Reply:
I would need to do that in the v2 push and it's already on my list. Other packages/projects depend on this package and so breaking the API without warning isn't OK.

Reply:
> I would need to do that in the v2 push

Meaning, a v2 of this package with breaking changes?

Reply:
Correct.